sql/moz-fx-data-shared-prod/telemetry_derived/main_1pct_v1/init.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE IF NOT EXISTS
+  `moz-fx-data-shared-prod.telemetry_derived.main_1pct_v1`
+PARTITION BY
+  DATE(submission_timestamp)
+CLUSTER BY
+  normalized_channel,
+  sample_id,
+  subsample_id
+OPTIONS
+  (require_partition_filter = TRUE, partition_expiration_days = 180)
```

Comment:
Should it be noted in the table description that this table has only the past 6 months of data?